ansible/roles/zuul/files/layout.yaml

```diff
@@ -354,6 +354,10 @@ projects:
   - name: Juniper/contrail-infra
     template:
       - name: ci-contrail-windows-other-projects-template
+      
+   - name: Juniper/contrail-deployers-containers
```

Comment:
Inconsistent indentation.
Do you know infra/update_zuul_layout/update-zuul-layout.py? It helps with updating layout.yaml.

Reply:
I was informed earlier  that update-zuul-layout.py was deprecated and pull req needs to be created manually .  Anyways i have fixed the indentation . please check